### PR TITLE
test: Add an e2e test for the COMPOSE_FILE environment variable in fi…

### PR DESCRIPTION
**Pull request by haytok**
…nch compose command

The following pull request attempts to modify finch compose command to allow the COMPOSE_FILE environment variable.

- https://github.com/runfinch/finch/pull/994

Therefore, this fix adds an e2e test to run finch compose command with the COMPOSE_FILE environment variable.

Issue #, if available: N/A

*Description of changes:* The details are described in this commit message.

*Testing done:* Yes

- [x] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: option/option.go [hunk 1]
```diff
@@ -50,3 +51,32 @@ func (o *Option) NewCmd(args ...string) *exec.Cmd {
 	cmd.Env = append(os.Environ(), o.env...)
 	return cmd
 }
+
+// UpdateEnv updates the environment variable for the key name of the input.
+func (o *Option) UpdateEnv(env string) {
```
Comment (pendo324) [inline option/option.go:56]: Would it be easier if the function signature took 2 arguments instead? Like:

```
func (o *Option) UpdateEnv(envKey string, envVal string) {
```

That way you don't have to split the string on `=` in `containsEnv`

**Review by pendo324 — COMMENTED** [review]
LGTM in principle, just left a suggestion. Thanks!

File: option/option.go (hunk 1 above)
Comment (haytok) [reply option/option.go:56]: Thanks for review!!! Will update.

Comment (haytok) [reply option/option.go]: fixed.

**Comment by haytok** [comment]
Thanks for review!!!, @pendo324 

I have fixed based on the review, so could you please re-review when you have time ???

**Review by pendo324 — APPROVED** [review]
LGTM, thanks!

**Comment by pendo324** [comment]
Next step is to create a new release of common-tests and bump the version in finch/finch-core. I can take care of that 👍

**Comment by haytok** [comment]
Thank you !!!

Also, sorry, I assigned the following PR .
Could you please review if the test failure in Windows environment is resolved 🙇

- https://github.com/runfinch/finch/pull/994

**Pull request merged**